tests/components/plugwise/test_config_flow.py

```diff
@@ -111,6 +111,34 @@ async def test_zeroconf_form(hass):
     assert len(mock_setup.mock_calls) == 1
     assert len(mock_setup_entry.mock_calls) == 1
 
+    result3 = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_ZEROCONF},
+        data=TEST_DISCOVERY,
+    )
+    assert result3["type"] == data_entry_flow.RESULT_TYPE_FORM
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)